dvc/repo/ls.py

```diff
@@ -110,6 +110,7 @@ def _ls(
             "isdir": info["type"] == "directory",
             "isexec": info.get("isexec", False),
             "size": info.get("size"),
+            "md5": dvc_info.get("md5"),
```

Comment:
As mentioned above, this could be `md5-dos2unix` too.

```suggestion
            "md5": dvc_info.get("md5") or dvc_info.get("md5-dos2unix"),
```